### PR TITLE
Hide legends if no legend is visible.

### PR DESCRIPTION
**Pull request by nobuti**
This PR hides the legends container if layer selector is not activated and no legend is visible.

map-legends-options

@alonsogarciapablo 🙏

### CODE REVIEW

**Comment by nobuti** [comment]
cc @xavijam

**Comment by xavijam** [comment]
Reviewing myself.

File: src/geo/ui/legends/layer-legends-view.js [hunk 1]
```diff
@@ -14,26 +15,40 @@ var LayerLegendsView = Backbone.View.extend({
     this._legendViews = [];
 
     this.settingsModel = options.settingsModel;
+    this.tryContainerVisibility = options.tryContainerVisibility;
```
Comment (xavijam) [inline src/geo/ui/legends/layer-legends-view.js]: What is it?

Comment (nobuti) [inline src/geo/ui/legends/layer-legends-view.js]: There is a general container for all layers. Then every layer has its own container for its legends. This general container is managed by a higher order view, so from the inner views we need to indicate a way to hide the general container.

Comment (xavijam) [inline src/geo/ui/legends/layer-legends-view.js]: Ajam!

**Pull request merged**